### PR TITLE
Fix ticdc canal-json related doc

### PR DESCRIPTION
**Pull request by yuqi1129**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: None
- Other reference link(s): None

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/troubleshoot-ticdc.md [hunk 1]
```diff
@@ -173,7 +173,7 @@ cdc cli changefeed create --pd=http://10.0.10.25:2379 --sink-uri="kafka://127.0.
 > **Note:**
 >
 > * This feature is introduced in TiCDC 4.0.2.
-> * TiCDC currently supports outputting data changes in the Canal format only to Kafka.
+> * TiCDC currently supports outputting data changes in the Canal format only to mq sink like Kafka, Pulsar.
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:176]:
```suggestion
> * TiCDC currently supports outputting data changes in the Canal format only to MQ sinks such as Kafka and Pulsar.
```

Comment (yuqi1129) [reply ticdc/troubleshoot-ticdc.md:176]: Good suggestion, i will follow your advice

**Review by amyangfei — APPROVED** [review]

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**